README.md

```diff
@@ -64,11 +64,25 @@ Remember to remove the k8 cluster to avoid ONS still being charged.
 
 ### Run - Local Locust against RH in GCP
 
+Before attempting a local run you'll need to create some test data and start the shell:
+
+	$ cd census-rh-performance-test
+	$ cp test_data/example_event_data.2.txt test_data/event_data.txt 
+	$ pipenv shell
+
 If you want to run a local Locust to generate traffic for a RH which is deployed in GCP then 
 you'll need a command like:
 
     $ locust -f locust_tasks/locustfile.py --no-web --clients 750 --hatch-rate 20 --csv-full-history --csv /tmp/rhui.csv --reset-stats --host=http://34.107.206.101
 
+To do a command line run against a local RH:
+	
+	$ locust -f locust_tasks/locustfile.py --no-web --clients 5 --hatch-rate 1 --csv-full-history --csv /tmp/rhui.csv --reset-stats --host http://localhost:9092 2>&1 | tee /tmp/locust.log
+
+To run in the browser firstly start Locust and then point the browser at the Locust control panel: http://localhost:8089/
+
+	$ locust -f locust_tasks/locustfile.py --host http://localhost:9092
```

Comment:
I get the same error (about importing TaskSequence) with both these ways of running it too.